swap_meet/decor.py

```diff
@@ -1,2 +1,15 @@
-class Decor:
-    pass
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, condition=None):
+        if condition == None:
+            super().__init__(category="Decor")
+        else:
+            super().__init__(condition = condition, category = "Decor")
```

Comment:
The style for passing named params or setting default values typically leaves out the spaces around the `=`. ([PEP8](https://www.python.org/dev/peps/pep-0008/#:~:text=Don't%20use%20spaces%20around%20the%20=%20sign))
```suggestion
            super().__init__(condition=condition, category="Decor")
```